### PR TITLE
Add Animal Crossing: Happy Home Designer

### PR DESCRIPTION
**Pull request by MelonSpeedruns**
Adds Animal Crossing: Happy Home Designer.

*Automatically generated by PR builder*

### CODE REVIEW

File: games/animal-crossing-happy-home-designer/game.dat [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+[[releases]]
+title = "000400000014F100"
+region = "usa"
+release_date = "2015-08-25"
+
+[[testcases]]
+title = "000400000014F100"
+compatibility = "0"
+date = "2017-11-08"
+version = "HEAD-0fe8d9f"
```
Comment (Schplee) [inline games/animal-crossing-happy-home-designer/game.dat]: Compatibility entries must be tested with the Nightly. Please update the `compatibility` field, screenshots, and wiki page accordingly.

File: games/animal-crossing-happy-home-designer/game.dat [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+cpu = "Intel Core i5-2400"
+gpu = "Geforce GTX 1050ti"
```
Comment (TheKoopaKingdom) [inline games/animal-crossing-happy-home-designer/game.dat]: Nit: This appears to be stylized as  "Geforce GTX 1050 Ti" on [Nvidia's site](https://www.nvidia.com/en-us/geforce/products/10series/geforce-gtx-1050/).

**Comment by acnleditor2** [comment]
freeze (except for lottie and goldie) (only with shader jit enabled)

**Comment by Chrisatsinnoh** [comment]
How do I download the fix that gets the game stuck in the intro?

**Comment by TheKoopaKingdom** [comment]
@Chrisatsinnoh This repository is not for Citra support. The best way to try out new changes is to, unless you're confident with Git, wait for the feature to be included in the Canary builds.

**Comment by TheKoopaKingdom** [comment]
@MelonSpeedruns please address the review comments, or this will be closed.

**Pull request closed without merging**